jdk8-slim/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM openjdk:8-jdk-slim
+
+CMD ["gradle"]
+
+ENV GRADLE_HOME /opt/gradle
+ENV GRADLE_VERSION 4.8
+
+RUN apt-get update && apt-get install -y --no-install-recommends wget unzip
```

Comment:
This should have `&& rm -rf /var/lib/apt/lists/*` at the end.  I'm also trying to decide whether it makes sense to purge these packages after the install is complete.  I noticed the Docker guys didn't do that in the base openjdk image.